frontend/main.js

```diff
@@ -118,8 +118,9 @@ async function startApp () {
           'chelonia/storeSecretKeys', 'chelonia/crypto/keyId',
           'chelonia/queueInvocation',
           'chelonia/contract/waitingForKeyShareTo',
+          'chelonia/contract/successfulKeySharesByContractID',
           'gi.actions/chatroom/leave',
-          'gi.actions/group/removeOurselves',
+          'gi.actions/group/removeMember',
```

Comment:
Before, contracts could not remove someone else from another group. Now they can.

Reply:
Good catch. Fixed it back.